sigs/addons/MIGRATION_TO_CORE.md

```diff
@@ -0,0 +1,32 @@
+# Migration From TF-Addons To TensorFlow Core
+
+### In-Progress & Previous Migrations:
+https://github.com/tensorflow/addons/projects/2/
+
+### Process 
```

Comment:
I think this could be quite complicated for cases like `ImageProjectiveTransformV2`.
As we could have an upstream request where the sponsor is already in the TF core team I don't think this double pass is plausible for this specific case.

Reply:
This seems to be helping external contribution for the migration only, not internal contribution for the migration

Reply:
@tanzhenyu Do you mean that you want a different process for Google internals?